cli.py

```diff
@@ -166,6 +166,10 @@ def fix_up(self):
     def extract(self):
         logging.info('Extracting scheduler module objs: %s', ' '.join(self.mod_objs))
         self.make(stage = 'collect', plugsched_tmpdir = self.tmp_dir, plugsched_modpath = self.mod_path)
+
+        self.plugsched_sh.cp(sym_vers,      self.work_dir, force=True)
+        self.plugsched_sh.cp(self.vmlinux,  self.work_dir, force=True)
```

Comment:
`vmlinux` is only used by cmd_init. I don't see why it should be a class member variable (self.vmlinux).
I think `sym_vers` and `vmlinux` should be passed to `extract` method as two parameters.
Like `self.extract(sym_vers, vmlinux)`

Reply:
After discussion, we decide `sym_vers`, `vmlinux`, `.config`, etc. all should be declared as class member variables.

And `cmd_init`, `cmd_build`, `cmd_dev_init` should all pass them as constructor parameters. Will open another  issue working on this.

Reply:
#42